rpcs3/Emu/Cell/Modules/cellSysutilAvc2.cpp

```diff
@@ -94,11 +99,12 @@ struct avc2_settings
 		save(ar);
 	}
 
-	void save(utils::serial& ar)
+	void save(utils::serial& ar) const
 	{
 		GET_OR_USE_SERIALIZATION_VERSION(ar.is_writing(), cellSysutil);
 
-		ar(avc2_cb, avc2_cb_arg, streaming_mode, mic_out_stream_sharing, video_stream_sharing, total_video_bitrate);
+		ar(avc2_cb, avc2_cb_arg, streaming_mode, mic_out_stream_sharing, video_stream_sharing, total_video_bitrate,
```

Comment:
@elad335 is there anything else to do for savestates here?

Reply:
Add a savestate module version check for these components.

Reply:
Done, I think